common/jersey-client/src/main/java/com/bazaarvoice/emodb/common/jersey/dropwizard/JerseyEmoClient.java

```diff
@@ -18,7 +17,6 @@
 
     public JerseyEmoClient(Client client) {
```

Comment:
consider to change to something like:
```suggestion
    public JerseyEmoClient(final Client client) {
        _client = checkNotNull(client, "client");
        if (!_client.getConfiguration().isRegistered(JacksonJsonProvider.class)) {
            _client.register(JacksonJsonProvider.class);
        }
    }
```

Reply:
updated with suggestion